src/commands/appdistribution-distribute.ts

```diff
@@ -98,11 +103,39 @@ module.exports = new Command("appdistribution:distribute <distribution-file>")
       );
     }
 
+    if (
+      distribution.distributionFileType() === DistributionFileType.AAB &&
+      app.aabState !== AabState.ACTIVE &&
+      app.aabState !== AabState.AAB_STATE_UNAVAILABLE
+    ) {
+      switch (app.aabState) {
+        case AabState.PLAY_ACCOUNT_NOT_LINKED: {
+          throw new FirebaseError("This project is not linked to a GooglePlay account.", {
```

Comment:
I know the copy isn't final but there should be a space between Google and Play

Reply:
Ahh my bad. I forgot to add the space there. Nice catch!

Reply:
All good! Updated!